tests: latency_measure: Increase verbosity of summary lines

peter-mitsis opened the pull request:
Feedback from the previous summary line change has been that it had become more difficult for people to parse and understand what is being tested/benchmarked when the description consisted of just using a terse tag. To improve the situation a more human oriented description has been added to follow that tag so that not only can tools still parse on the tag, but people can use both tag and the extra description to understand the test/benchmark.

Summary lines for each test now consist of the following:
    1. A terse tag (for tools to parse)
    2. A more human oriented description.
    3. Number of cycles.
    4. Number of nanoseconds.

peter-mitsis committed:
tests: latency_measure: Increase verbosity of summary lines

Feedback from the previous summary line change has been that it
had become more difficult for people to parse and understand what
is being tested/benchmarked when the description consisted of
just using a terse tag. To improve the situation a more human
oriented description has been added to follow that tag so that
not only can tools still parse on the tag, but people can use
both tag and the extra description to understand the test/benchmark.

Summary lines for each test now consist of the following:
    1. A terse tag (for tools to parse)
    2. A more human oriented description.
    3. Number of cycles.
    4. Number of nanoseconds.

Signed-off-by: Peter Mitsis <peter.mitsis@intel.com>

nashif commented:
just convert something like this:

`ISR resume.interrupted.thread.kernel     - Return from ISR to interrupted thread              :     353 cycles ,     2949 ns`

to 

`isr.resume.interrupted.thread.kernel     - Return from ISR to interrupted thread              :     353 cycles ,     2949 ns`

nashif commented:
also, I would change the strings

`take.blocking.(K -> K)`

to 

`take.blocking.k_to_k`

That is easier to read and parse eventually.   `blah.(K -> K)` looks weird.

peter-mitsis committed:
tests: latency_measure: Remove prj_user.conf

Moves the enablement of userspace from the project configuration
file to testcase.yaml.

Signed-off-by: Peter Mitsis <peter.mitsis@intel.com>

peter-mitsis committed:
tests: latency_measure: tweak frdm_k64f config

The FRDM K64F board runs at 120 MHz and has a 24 bit timer.
According to its timer driver, its MAX_TICKS (which comes into
play for a tickless kernel) is calculated as ...
     CYC_PER_TICK = 120 MHz / ticks per second
     MAX_TICKS = (COUNTER_MAX / CYC_PER_TICK) - 1

To achieve a minimum value of MAX_TICKS=1, ticks per second must
be set to at least 15. Experimentation has shown that setting the
number of ticks per second below this threshhold on a tickless
kerenl leads to unreliable timestamps.

Signed-off-by: Peter Mitsis <peter.mitsis@intel.com>
